app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/movielist/viewmodel/ListViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.marcinwisniewski.intivetrainingapp.movielist.viewmodel
+
+import android.arch.lifecycle.ViewModel
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.MovieProvider
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.MovieResponse
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+class ListViewModel : ViewModel() {
```

Comment:
Why this class inherits from ViewModel() if there is only public method to return an Observable? 
Try to learn more about MVVM pattern.